packages/foundations/scripts/color-placeholders-generator.js

```diff
@@ -113,14 +124,15 @@ const generateBGVariants = (
 
     &-ia, &[data-variant="interactive"] {
 		@extend %${placeholderName};
-		&:enabled{
-			&:hover{
-				--${prefix}-bg-transparent: 84%;
-			}
-			&:active{
-				--${prefix}-bg-transparent: 68%;
+		&:is(#{$interactive-elements}) {
+			&:enabled{
+				@extend %interactive-states;
 			}
 		}
+
+		&:is(a) {
+			@extend %interactive-states;
+		}
```

Comment:
```suggestion
		&:is(a),:is(#{$interactive-elements}):enabled{
			@extend %interactive-states;
		}
```

why couldn't you merge this with the other selector above?